tests/h/util/metrics_test.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+import pytest
+from mock import call
+from h.util import metrics
+
+from webob.multidict import MultiDict
+
+
+class TestSendMetric(object):
+
+    def test_send_metric(self, newrelic):
+        metrics.send_metric("testkey", "testvalue")
+        newrelic.agent.add_custom_parameter.assert_called_once_with("testkey", "testvalue")
+
+    @pytest.fixture(autouse=True)
+    def newrelic(self, patch):
+        return patch("h.util.metrics.newrelic")
+
+
+@pytest.mark.usefixtures("send_metric")
+class TestRecordSearchApiUsageMetrics(object):
+
+    def test_records_parameters(self, send_metric):
+        md = MultiDict(tags="tagsvalue", url="urlvalue", bad="unwanted")
```

Comment:
Maybe `params` would be a better name for this variable, as that's what the argument is called in `record_search_api_usage_metrics()`?